pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -449,8 +472,30 @@ func (f *ring0Factory) ResolveImage(name string) (string, error) {
 }
 
 func (f *ring0Factory) Resumer(info *resource.Info) ([]byte, error) {
+	_, isUnstructured := info.Object.(runtime.Unstructured)
+	if isUnstructured {
```

Comment:
nit: what if `info.Object` is not Unstructured, then it will go to the switch default which is `return nil, fmt.Errorf("resuming is not supported or info.Object is not runtime.Unstructured")` ?